src/modules/usrloc/README

```diff
@@ -1001,7 +1001,9 @@ modparam("usrloc", "rm_expired_delay", 30)
 
 3.47. server_id_filter (int)
 
-   Enable (1) or disable (0) filter records by server_id on load.
+   Enable (1) or disable (0) filter records by server_id on load and during cleaning of expired db records.
```

Comment:
You should not change README directly
https://www.kamailio.org/wiki/devel/git-commit-guidelines#generic_guidelines